@here/harp-mapview/lib/geometry/TileGeometryCreator.ts

```diff
@@ -729,15 +729,16 @@ export class TileGeometryCreator {
                     object.onBeforeRender = chainCallbacks(
```

Comment:
Why registering this callback at all if globe projection is active?

Reply:
Good question, maybe if the projection changes at runtime. But if this is something we don't need to support, then yes, all good.

In saying that though, are the objects recreated if the projection changes? I guess so, so yes, this would make sense.

Reply:
yes, if the projection changes we clear the tile cache, so no need for runtime change